Even with removed patch, building fails with 300 000 lines of log.
```
/nix/store/lbk30k56awz9vz9qpid93fkjns0xwlhd-gcc-wrapper-13.3.0/bin/g++ -DBWE_TEST_LOGGING_COMPILE_TIME_ENABLE=0 -DHAVE_WEBRTC_VIDEO -DNDEBUG -DNO_MAIN_THREAD_WRAPPING -DQT_DEPRECATED_WARNINGS_SINCE=0x051500 -DQT_IMPLICIT_QCHAR_CONSTRUCTION -DQT_NO_CAST_FROM_BYTEARRAY -DQT_NO_KEYWORDS -DRTC_DISABLE_TRACE_EVENTS -DRTC_ENABLE_H265 -DRTC_ENABLE_VP9 -DTGCALLS_USE_STD_OPTIONAL -DWEBRTC_APM_DEBUG_DUMP=0 -DWEBRTC_APP_TDESKTOP -DWEBRTC_DUMMY_AUDIO_BUILD -DWEBRTC_ENABLE_PROTOBUF=0 -DWEBRTC_HAVE_DCSCTP -DWEBRTC_HAVE_SCTP -DWEBRTC_INCLUDE_INTERNAL_AUDIO_DEVICE -DWEBRTC_LIBRARY_IMPL -DWEBRTC_LINUX -DWEBRTC_NON_STATIC_TRACE_EVENT_HANDLERS=1 -DWEBRTC_OPUS_SUPPORT_120MS_PTIME=1 -DWEBRTC_OPUS_VARIABLE_COMPLEXITY=0 -DWEBRTC_POSIX -DWEBRTC_USE_BUILTIN_ISAC_FLOAT -DWEBRTC_USE_H264 -DWEBRTC_USE_PIPEWIRE -DWEBRTC_USE_X11 -I/build/source/Telegram/ThirdParty/tgcalls -I/build/source/Telegram/ThirdParty/tgcalls/tgcalls -isystem /nix/store/2bw54660ndcc79c6zjm5g98slkpks814-tg_owt-0-unstable-2024-08-02-dev/include/tg_owt -isystem /nix/store/2bw54660ndcc79c6zjm5g98slkpks814-tg_owt-0-unstable-2024-08-02-dev/include/tg_owt/third_party/libyuv/include -O3 -DNDEBUG -std=gnu++20 -Wno-deprecated-volatile -Wno-ambiguous-reversed-operator -Wno-deprecated-declarations -Wno-unqualified-std-cast-call -fPIC -fno-strict-aliasing -pipe -Wall -Wextra -Wno-unused-parameter -Wno-switch -Wno-maybe-uninitialized -Wno-missing-field-initializers -Wno-sign-compare -Wno-deprecated -MD -MT Telegram/CMakeFiles/lib_tgcalls.dir/ThirdParty/tgcalls/tgcalls/LogSinkImpl.cpp.o -MF Telegram/CMakeFiles/lib_tgcalls.dir/ThirdParty/tgcalls/tgcalls/LogSinkImpl.cpp.o.d -o Telegram/CMakeFiles/lib_tgcalls.dir/ThirdParty/tgcalls/tgcalls/LogSinkImpl.cpp.o -c /build/source/Telegram/ThirdParty/tgcalls/tgcalls/LogSinkImpl.cpp
In file included from /build/source/Telegram/ThirdParty/tgcalls/tgcalls/LogSinkImpl.cpp:3:
/build/source/Telegram/ThirdParty/tgcalls/tgcalls/Instance.h:17:7: error: conflicting declaration of template 'template<class T> struct rtc::scoped_refptr'
   17 | class scoped_refptr;
      |       ^~~~~~~~~~~~~
In file included from /nix/store/2bw54660ndcc79c6zjm5g98slkpks814-tg_owt-0-unstable-2024-08-02-dev/include/tg_owt/rtc_base/checks.h:59,
``` 

[log](https://files.catbox.moe/pwkyrd.log)

--- a/pkgs/by-name/ma/materialgram/package.nix
+++ b/pkgs/by-name/ma/materialgram/package.nix
@@ -1,0 +1,34 @@
+{
+  telegram-desktop,
+  lib,
+  fetchFromGitHub,
+}:
+telegram-desktop.overrideAttrs (
+  finalAttrs: previousAttrs: {
+    pname = "materialgram";
+    version = "5.4.1.1";
+
+    src = fetchFromGitHub {
+      owner = "kukuruzka165";
+      repo = "materialgram";
+      rev = "refs/tags/v${finalAttrs.version}";
+      fetchSubmodules = true;
+      hash = "sha256-enA/8mIXcRTPGyZFgZ5Wg4fWq4fD4rtjH7bm/32FZaE";
+    };
+
+    meta = previousAttrs.meta // {
+      description = "Telegram Desktop fork with material icons and some improvements";
+      longDescription = ''
+        Telegram Desktop fork with Material Design and other improvements,
+        which is based on the Telegram API and the MTProto secure protocol.
+      '';
+      homepage = "https://kukuruzka165.github.io/materialgram/";
+      changelog = "https://github.com/kukuruzka165/materialgram/releases/tag/v${finalAttrs.version}";
+      maintainers = with lib.maintainers; [
+        oluceps
+        aleksana
+      ];
+      mainProgram = "materialgram";
+    };
+  }
+)

